### ISSUE TITLE
[CI] FiltersAggregatorTests testDocValuesFieldExistsForKeyword failing

### ISSUE THREAD

**Issue by nik9000** [elastic/elasticsearch#73071]
**Build scan:**
https://gradle-enterprise.elastic.co/s/tldpneiacy4ay/tests/:server:test/org.elasticsearch.search.aggregations.bucket.filter.FiltersAggregatorTests/testDocValuesFieldExistsForKeyword

**Reproduction line:**
`./gradlew ':server:test' --tests "org.elasticsearch.search.aggregations.bucket.filter.FiltersAggregatorTests.testDocValuesFieldExistsForKeyword" -Dtests.seed=A796A4F96C5EAD32 -Dtests.locale=nn-NO -Dtests.timezone=Europe/London -Druntime.java=8 -Dtests.fips.enabled=true`

**Applicable branches:**
7.x

**Reproduces locally?:**
Yes

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.search.aggregations.bucket.filter.FiltersAggregatorTests&tests.test=testDocValuesFieldExistsForKeyword

**Failure excerpt:**
```
java.lang.AssertionError: 
Expected: <0>
     but: was <1>

  at __randomizedtesting.SeedInfo.seed([A796A4F96C5EAD32:A585FBE9A1007CE6]:0)
  at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
  at org.junit.Assert.assertThat(Assert.java:956)
  at org.junit.Assert.assertThat(Assert.java:923)
  at org.elasticsearch.search.aggregations.bucket.filter.FiltersAggregatorTests.lambda$docValuesFieldExistsTestCase$45(FiltersAggregatorTests.java:954)
  at org.elasticsearch.search.aggregations.AggregatorTestCase.debugTestCase(AggregatorTestCase.java:639)
  at org.elasticsearch.search.aggregations.AggregatorTestCase.lambda$debugTestCase$9(AggregatorTestCase.java:593)
  at org.elasticsearch.search.aggregations.AggregatorTestCase.withIndex(AggregatorTestCase.java:557)
  at org.elasticsearch.search.aggregations.AggregatorTestCase.debugTestCase(AggregatorTestCase.java:593)
  at org.elasticsearch.search.aggregations.bucket.filter.FiltersAggregatorTests.docValuesFieldExistsTestCase(FiltersAggregatorTests.java:941)
  at org.elasticsearch.search.aggregations.bucket.filter.FiltersAggregatorTests.testDocValuesFieldExistsForKeyword(FiltersAggregatorTests.java:909)
  at sun.reflect.NativeMethodAccessorImpl.invoke0(NativeMethodAccessorImpl.java:-2)
  at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  at java.lang.reflect.Method.invoke(Method.java:498)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1750)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:938)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:974)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:988)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:49)
  at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
  at org.apache.lucene.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:48)
  at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
  at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:368)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:817)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:468)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:947)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:832)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:883)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:894)
  at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:41)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
  at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
  at org.apache.lucene.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:54)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:368)
  at java.lang.Thread.run(Thread.java:748)

```

**Comment by nik9000** [comment]
This one's mine.

**Comment by astefan** [comment]
Another one https://gradle-enterprise.elastic.co/s/ciyafxgrnrjly.
```
04:18:30 org.elasticsearch.search.aggregations.bucket.filter.FiltersAggregatorTests > testDocValuesFieldExistsForKeyword FAILED
04:18:30     java.lang.AssertionError: 
04:18:30     Expected: <0>
04:18:30          but: was <2>
04:18:30         at __randomizedtesting.SeedInfo.seed([5208368395EEDF1C:501B699358B00EC8]:0)
04:18:30         at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
04:18:30         at org.junit.Assert.assertThat(Assert.java:956)
04:18:30         at org.junit.Assert.assertThat(Assert.java:923)
04:18:30         at org.elasticsearch.search.aggregations.bucket.filter.FiltersAggregatorTests.lambda$docValuesFieldExistsTestCase$45(FiltersAggregatorTests.java:954)
04:18:30         at org.elasticsearch.search.aggregations.AggregatorTestCase.debugTestCase(AggregatorTestCase.java:639)
04:18:30         at org.elasticsearch.search.aggregations.AggregatorTestCase.lambda$debugTestCase$9(AggregatorTestCase.java:593)
04:18:30         at org.elasticsearch.search.aggregations.AggregatorTestCase.withIndex(AggregatorTestCase.java:557)
04:18:30         at org.elasticsearch.search.aggregations.AggregatorTestCase.debugTestCase(AggregatorTestCase.java:593)
04:18:30         at org.elasticsearch.search.aggregations.bucket.filter.FiltersAggregatorTests.docValuesFieldExistsTestCase(FiltersAggregatorTests.java:941)
04:18:30         at org.elasticsearch.search.aggregations.bucket.filter.FiltersAggregatorTests.testDocValuesFieldExistsForKeyword(FiltersAggregatorTests.java:909)
04:18:31 REPRODUCE WITH: ./gradlew ':server:test' --tests "org.elasticsearch.search.aggregations.bucket.filter.FiltersAggregatorTests.testDocValuesFieldExistsForKeyword" -Dtests.seed=5208368395EEDF1C -Dtests.locale=hi -Dtests.timezone=America/Argentina/ComodRivadavia -Druntime.java=8
```

**Comment by nik9000** [comment]
This'd be me but I'm away today. If it's causing trouble please mute it.

**Comment by nik9000** [comment]
Duplicate of #73185.

**Comment by nik9000** [comment]
Closed by #73336
